auth/src/lib.rs

```diff
@@ -188,3 +235,54 @@ pub fn authenticate(profiles: &[profile::Profile], options: Options) -> anyhow::
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use anyhow::Result;
+    use temp_env;
+    use tempdir::TempDir;
+
+    use super::*;
+
+    const PASSWORD: &str = "pass";
+
+    fn create_auth_options(username: &str) -> Options {
+        Options {
+            active: false,
+            init: true,
+            username: Some(username.to_owned()),
+            password: Some(PASSWORD.to_owned())
+        }
+    }
+
+    fn cleanup() -> Result<()> {
+        for profile in profile::list()? {
+            let pass = term::pwhash(SecUtf8::from(PASSWORD));
+            keys::remove(&profile, pass, keys::ssh_auth_sock())?;
+        }
+        Ok(())
+    }
+
+    #[test]
+    fn profile_can_be_initialized() -> Result<()> {
+        let home_dir = TempDir::new("lnk-home")?;
+        let home = home_dir.path().to_str().unwrap();
+        
+        temp_env::with_var("LNK_HOME", Some(home), || {
```

Comment:
You can use `librad::profile::LNK_HOME` here and below to make the code a little more robust